addons/website/models/__init__.py

```diff
@@ -26,3 +26,4 @@
 from . import website_visitor
 from . import website_snippet_filter
 from . import website_configurator_feature
+from . import website_form
```

Comment:
alphabetical please, you can also reorder the last 3 before your line.. lets keep things clean

```
from . import website_configurator_feature
from . import website_form
from . import website_snippet_filter
from . import website_visitor
```